charts/osm-arc/values.yaml

```diff
@@ -9,6 +9,7 @@ Azure:
 
 OpenServiceMesh:
     ignoreNamespaces: "kube-system azure-arc arc-osm-system"
+    validateNamespaces: "kube-system osm-system"
```

Comment:
Could this key be named something a bit more intuitive? I feel like outside of the `validate-single-mesh` spec I wouldn't know what it is

Reply:
I actually don't need this field anymore so I'm getting rid of it.